component/service/src/main/java/org/exoplatform/social/rest/suggest/SpaceRestServices.java

```diff
@@ -315,11 +315,15 @@ public Response leave(@ApiParam(value = "Space technical identifier", required =
         if (space == null) {
           return Response.status(HTTPStatus.NOT_FOUND).build();
         }
-  
+        if (spaceService.isManager(space, userId)) {
+          spaceService.setManager(space, userId, false);
+        }
```

Comment:
IMO, it would be better to add this business logic in Service layer as recommended here : https://community.exoplatform.com/portal/g/:spaces:engineering/exo_architects/notes/13754

Reply:
done